docs/userGuide/syntax/textStyles.mbdf

```diff
@@ -26,7 +26,7 @@ Syntax added by MarkBind:
 <include src="codeAndOutput.md" boilerplate >
 <variable name="highlightStyle">markdown</variable>
 <variable name="code">
-****Super Bold****, ++Underline++, ==Highlight==, %%Dim%%, Super^script^, Sub~script~
+****Super Bold****, $$Underline$$, ==Highlight==, %%Dim%%, ++Large++, --Small--, Super^script^, Sub~script~
```

Comment:
Users that are already used to use `$$` for underlining might not catch that the new syntax is `++`. Should we note in the docs that there's a change from this version onwards?

Reply:
Hmm good point. On second thought, it might not be good to change the syntax for underlining straightaway. However, it seems it won't be easy to introduce some form of backward compatibility for this new feature. 

Do you think it would be better to introduce some form of intermediary change (e.g. support both `++` and `$$` as underline and use another syntax for larger text for in the meantime, before we fully deprecate `++` for underlining and use it for larger text)?

@damithc would you have any advice for this?

Reply:
> @damithc would you have any advice for this?

As I'm the only user as of now, just let me know when the change happens so that I can migrate the code when I upgrade the version.

Reply:
I'll be able to pull _just_ the `$$...$$` syntax into another branch separately and get a new release out with just that once this is done; but a PR against the branch would be welcome as well 🙂 https://github.com/MarkBind/markbind/tree/v2.7.x-hotfixes

Reply:
> I'll be able to pull _just_ the `$$...$$` syntax into another branch separately and get a new release out with just that once this is done; but a PR against the branch would be welcome as well 🙂 

Have made the intermediary PR here :-)

https://github.com/MarkBind/markbind/pull/1476